### ISSUE TITLE
v0.13 Installation failed - Ubuntu 14.04 & 16.04

### ISSUE THREAD

**Issue by vClouds** [wekan/wekan#908]
After struggling with node installation of n 0.10.48 and adnvces 14.04 with libstdc++6
the below error were obtained on both,

> /var/www/wekan/bundle/programs/server/node_modules/fibers/future.js:280
> 						throw(ex);
> 						      ^
> Error: MONGO_URL must be set in environment
>     at Object.<anonymous> (packages/mongo/remote_collection_driver.js:36:11)
>     at Object.defaultRemoteCollectionDriver (packages/underscore/underscore.js:750:1)
>     at new Mongo.Collection (packages/mongo/collection.js:103:40)
>     at Package (packages/idmontie_migrations/migrations.js:23:1)
>     at packages/idmontie_migrations/migrations.js:237:1
>     at packages/idmontie_migrations/migrations.js:237:1
>     at /var/www/wekan/bundle/programs/server/boot.js:297:10
>     at Array.forEach (native)
>     at Function._.each._.forEach (/var/www/wekan/bundle/programs/server/node_modules/underscore/underscore.js:79:11)
>     at /var/www/wekan/bundle/programs/server/boot.js:133:5

Please note that, mongo 3.2.11 were installed and the below export commands were done as well,

> export MONGO_URL='mongodb://127.0.0.1:27017/admin'
> export ROOT_URL='http://localhost:3000'
> export MAIL_URL='smtp://user:pass@mailserver.example.com:25/'
> export PORT=3000

Please advise on v0.13 install on any 14.04 or 16.04
One more thing, please clarify on port, such manual showing 8080

**Comment by froiloc** [comment]
I installed it today on Ubuntu 16.04. Had to install node 0.10.41. I set the environment variables in ~/.bashrc at the end of the file. This worked. Though I cannot send emails. But that's another issue.

Nevertheless, in your case the user that runs _node main.js_ must have these environment variables set. Test it before starting the wekan server by using the _export_ command in shell. This will list all environment variables.

**Comment by xet7** [comment]
Environment variables are documented at:
https://github.com/wekan/wekan/wiki/Install-and-Update#run-wekan-manually-if-youve-built-for-production

If you watnt port 8080, you need to change it at line 2 and 4:
```
export MONGO_URL='mongodb://127.0.0.1:27017/admin'
export ROOT_URL='http://localhost:8080'
export MAIL_URL='smtp://user:pass@mailserver.example.com:25/'
export PORT=8080
```

**Comment by JeremiasKnapp** [comment]
@vClouds what was your solution for the problem with libstdc++6 ?
I think i'm struggling with the same problem.
My error message:
> Error: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by /root/bundle/programs/server/npm/node_modules/meteor/npm-bcrypt/node_modules/bcrypt/build/Release/bcrypt_lib.node)

**Comment by xet7** [comment]
Install todays release v0.19, it has been built on Ubuntu 14.04.
